src/Microsoft.AspNet.WebSockets.Protocol/CommonWebSocket.cs

```diff
@@ -592,11 +592,11 @@ private void ValidateSegment(ArraySegment<byte> buffer)
             }
             if (buffer.Offset < 0 || buffer.Offset > buffer.Array.Length)
             {
-                throw new ArgumentOutOfRangeException(nameof(buffer.Offset), buffer.Offset, string.Empty);
+                throw new ArgumentOutOfRangeException(nameof(buffer), buffer.Offset, string.Empty);
```

Comment:
That looks the same as what I said, no? BTW don't use string interpolation here because once these go into RESX files you can't use string interpolation - so stick to `string.Format`.